components/nexus-security/src/test/java/org/sonatype/nexus/security/AbstractSecurityTest.java

```diff
@@ -47,6 +49,7 @@
 public abstract class AbstractSecurityTest
 {
   protected final TestUtil util = new TestUtil(this);
+  protected final String ehCacheConfigPath = "/ehcache-test.xml";
```

Comment:
In case the concrete unit test needs/wants to override the path to the configuration.